lib/serviceClient.ts

```diff
@@ -12,7 +12,7 @@ import { isStreamOperation, OperationSpec } from "./operationSpec";
 import { deserializationPolicy, DeserializationContentTypes } from "./policies/deserializationPolicy";
 import { exponentialRetryPolicy } from "./policies/exponentialRetryPolicy";
 import { generateClientRequestIdPolicy } from "./policies/generateClientRequestIdPolicy";
-import { msRestUserAgentPolicy } from "./policies/msRestUserAgentPolicy";
+import { userAgentPolicy } from "./policies/telemetry/userAgentPolicyFactory";
```

Comment:
Can you figure out a way to make this import the same as the other policy imports instead of having the `Factory` suffix?